helm/stargate/values.yaml

```diff
@@ -0,0 +1,42 @@
+# Default values for stargate.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
```

Comment:
Consider adding comments for chart properties. I consider values.yaml to be somewhat of an API of sorts so having it well documented is critical IMO.

Reply:
Added comments in the values.yaml file, Also added readMe file